### PR TITLE
5 weme

### PR DESCRIPTION
**Pull request by chphoom**
this pull req changes the implementation of the challenge database and entities

it now stores we and me challenges and the related extra inforrmation

using the fastapi docs to test methods may not result in expected behavior because the json information passed does not include any typing, however using resetdb all functions work as expected and the database can be updated with the correct information.

reset db now loads a starting wechallenge. 

future TODOs will include whatever functions are necessary to update particular fields such as saving a challenge by a user, likewise to the example many of these rely on other things being implemented first so we will address them as it comes

### CODE REVIEW

**Comment by chphoom** [comment]
please merge the pull req #26 before this one, and i will handle merge conflicts

File: backend/script/reset_db.py [hunk 1]
```diff
@@ -10,7 +10,33 @@
...
+    user_entity: entities.UserEntity = entities.UserEntity.from_model(
+        models.User(
+            email="keaw@email.unc.com", 
+            displayName="keaw",
+            password="4ho00o76i1", 
+            created=datetime.now(), 
+            private=True, 
+            bio="", 
+            pronouns="they/them", 
+            img="",
```
Comment (elaine331917) [inline backend/script/reset_db.py:23]: maybe change this attribute name to avatar or pfp for clarity

File: backend/entities/challenge_entity.py [hunk 2]
```diff
@@ -29,7 +32,10 @@ def from_model(cls, model: Challenge) -> Self:
             adj=model.adj, 
             emotion=model.emotion, 
             style=model.style, 
-            colors=model.colors
+            colors=model.colors,
+            createdBy=model.createdBy,
+            start=model.start,
+            end=model.end
```
[the remark this answers is not in the document]
Comment (chphoom) [reply backend/entities/challenge_entity.py]: see commit [ffae385](https://github.com/chphoom/aroundtheblock/pull/27/commits/ffae385f6748ed228d7d53f507de27a52d1f0703)

File: backend/script/reset_db.py (hunk 1 above)
Comment (chphoom) [reply backend/script/reset_db.py]: see [ffae385](https://github.com/chphoom/aroundtheblock/pull/27/commits/ffae385f6748ed228d7d53f507de27a52d1f0703)

**Pull request merged**